lib/ClangImporter/ClangImporter.cpp

```diff
@@ -959,8 +995,22 @@ bool ClangImporter::importBridgingHeader(StringRef header, Module *adapter,
                                          SourceLoc diagLoc,
                                          bool trackParsedSymbols,
                                          bool implicitImport) {
-  if (llvm::sys::path::extension(header).endswith(PCH_EXTENSION)) {
-    return importBridgingPCH(header, adapter);
+  if (Impl.IsReadingBridgingPCH) {
+    // We already imported this with -include-pch above, so we should have
+    // collected a bunch of PCH-read modules that we need to finish
+    // processing.
+    for (auto CM : Impl.PCHReadModules) {
+      auto L = nextSyntheticImportLocation();
+      getClangPreprocessor().makeModuleVisible(CM, L);
```

Comment:
Shouldn't it already be visible thanks to the PCH? You may be able to get away with just calling `finishLoadingClangModule`.